stories/create-stories-from-doc.js

```diff
@@ -16,22 +16,21 @@ function createDirectoryIfNeeded(path) {
   }
 }
 
-function createMDXDocumentation(content) {
-  return `import { Story, Canvas } from '@storybook/addon-docs/blocks';\n\n${content}`
-}
-
-function createTemplate(component, index, content) {
-  return `import CustomMDXDocumentation from './Custom-MDX-Documentation.mdx';\n\
-  export default {\n\
+function createTemplate(component) {
+  return `export default {\n\
     title: 'Components/${component}',\n\
```

Comment:
I'd say yes to have the same links as https://boosted.orange.com/storybook/?path=/docs/components-accordion--docs to avoid having to update all the integrations in ZeroHeight.